packages/docx/src/ComponentStatus/ComponentStatus.tsx

```diff
@@ -76,7 +76,7 @@ export function ComponentStatus({
   };
 
   return (
-    <div>
+    <div style={{ display: "none" }}>
```

Comment:
I'm wondering if instead of returning `display: none`. We should just return a [fragment](https://reactjs.org/docs/fragments.html) `<></>` for now so that we are totally removing this element from the DOM.

Thoughts?

Reply:
Oh that make sense! Didn't know that was a thing 😄

Reply:
@eddysims after our offline discussion about the larger changes this requires, going to leave as-is